### PR TITLE
add norm op（cross_channel_norm）

### LINKED ISSUE #6561: CrossChannelNorm Operator.

### PR DESCRIPTION
**Pull request by sweetsky0901**
fix #6561

### CODE REVIEW

File: paddle/operators/norm_op.h [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+#pragma once
+#include "paddle/framework/op_registry.h"
+#include "paddle/operators/math/math_function.h"
+
+namespace paddle {
+namespace operators {
+
+template <typename T, int MajorType = Eigen::RowMajor,
+          typename IndexType = Eigen::DenseIndex>
+using EigenVector = framework::EigenVector<T, MajorType, IndexType>;
```
Comment (wanghaox) [inline paddle/operators/norm_op.h]: no using in .h files.

File: paddle/operators/norm_op.h [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+template <typename T, int MajorType = Eigen::RowMajor,
+          typename IndexType = Eigen::DenseIndex>
+using EigenMatrix = framework::EigenMatrix<T, MajorType, IndexType>;
```
Comment (wanghaox) [inline paddle/operators/norm_op.h]: no using in .h files.

Comment (sweetsky0901) [reply paddle/operators/norm_op.h]: done

File: paddle/operators/norm_op.h (hunk 1 above)
Comment (sweetsky0901) [reply paddle/operators/norm_op.h]: done

File: paddle/operators/norm_op.h [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+    framework::Tensor x_square;
+    x_square.mutable_data<T>(in_x->dims(), context.GetPlace());
+    auto x_square_eigen =
+        framework::EigenMatrix<T, Eigen::RowMajor, Eigen::DenseIndex>::From(
+            x_square, framework::make_ddim({batch_size, fea_len * channels}));
+    x_square_eigen.device(*place) = x.square();
+    auto scale_eigen =
+        framework::EigenVector<T, Eigen::RowMajor, Eigen::DenseIndex>::Flatten(
+            *scale);
+    for (int n = 0; n < batch_size; ++n) {
```
Comment (emailweixu) [inline paddle/operators/norm_op.h]: It seems to me that there is no need to use loop. We can do it for the whole batch together.

**Pull request merged**